package.json

```diff
@@ -9,9 +9,10 @@
   },
   "author": "",
   "license": "ISC",
+  "type": "module",
   "dependencies": {
-    "@bandwidth/voice": "^1.5.0",
-    "@bandwidth/webrtc": "^0.4.1",
+    "@bandwidth/voice": "^2.0.1",
+    "@bandwidth/webrtc": "file:../node-webrtc",
```

Comment:
Will update this once https://github.com/Bandwidth/node-webrtc/pull/10 has been merged